ogcserver/wms111.py

```diff
@@ -166,6 +166,19 @@ def GetCapabilities(self, params):
                 rootlayercrs.text = epsgcode.upper()
                 rootlayerelem.append(rootlayercrs)
 
+            for epsgcode in self.allowedepsgcodes:
```

Comment:
@pthorin just curious about why this is not in the previous `for` loop.

Reply:
Fair question. I don't see any reason why they can't be made into one single loop.